test/test_run_analysis.rb

```diff
@@ -71,7 +71,6 @@ def test_errors_already_exists
     yml = ' -y test/tests_yml_files/yml_bad_value/testing_baseline.yml'
     @command += yml
 
-    cli_output = `#{@command}`
```

Comment:
@shorowit This was intentionally the same line repeated. Is there a way to tell rubocop (or whatever we're using) to ignore this one?

Reply:
Aha, thanks, I can address it then. It didn't dawn on me that the failing test could be the result of a change here.

Reply:
To be clear, rubocop isn't complaining that the same line is repeated, it's complaining the the `cli_output` in the first line is unused (because it is overridden by the following line). So the first line just needs to be `#{@command}` and then rubocop will be happy and the CI test will be happy.